muikku-rest/src/main/java/fi/muikku/rest/user/UserRESTService.java

```diff
@@ -70,18 +82,62 @@ public Response searchUsers(
 			@QueryParam("searchString") String searchString,
 			@QueryParam("firstResult") @DefaultValue("0") Integer firstResult,
 			@QueryParam("maxResults") @DefaultValue("10") Integer maxResults,
+			@QueryParam("userGroupIds") List<Long> userGroupIds,
+      @QueryParam("myUserGroups") Boolean myUserGroups,
+			@QueryParam("workspaceIds") List<Long> workspaceIds,
+      @QueryParam("myWorkspaces") Boolean myWorkspaces,
 			@QueryParam("archetype") String archetype) {
 	  
 	  if (!sessionController.isLoggedIn()) {
 	    return Response.status(Status.FORBIDDEN).build();
 	  }
 
+	  UserEntity loggedUser = sessionController.getLoggedUserEntity();
```

Comment:
if user tries to request both myUserGroups and userGroupIds bad request should be returned